library-best-practices/JLBP-9.md

```diff
@@ -1,20 +1,21 @@
 [JLBP-9] Support the minimum Java version of your consumers
 -----------------------------------------------------------
 
-If a library B requiring Java 1.7 depends on a library A requiring Java 1.7, and
-library A bumps its minimum version to Java 1.8, then library B cannot pick up
-the new version of library A until it also bumps its own minimum Java
-version. This can result in a situation where the ecosystem is stuck on older
-library versions, preventing the pickup of important things like security fixes,
-and it can also lead to diamond dependency conflicts where no version works for
-all consumers. Thus, libraries should make sure their actively-maintained
-consumers advance their minimum version of Java before they advance it
-themselves. (Waiting for every last consumer could block an upgrade permanently,
-due to cases like abandoned projects.)
+Imagine that library B depends on library A and both work with Java 8.
+If library A releases a new version that requires Java 11, library B cannot upgrade
+to the new version of library A unless it also starts requiring Java 11. 
+If library B is not able to do that—for instance because it must work in an environment
+such as App Engine that does not yet support Java 11—it must keep using the older
+version of library A, missing out on security and bug fixes in the newer version.
```

Comment:
Just a suggestion.

```suggestion
version of library A, missing out on security and bug fixes in its newer versions.
```

to emphasize the effect is not just one version but other releases in future.